bin/add_exps_to_collection.sh

```diff
@@ -3,7 +3,6 @@
 set -e
 
 scriptDir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-source $scriptDir/db_scxa_common.sh
 
```

Comment:
Should't you have here replacing the line you've removed? 

```suggestion
source $scriptDir/common_routines.sh
```

Reply:
no, as we are not using any functions from that shared file